snapcraft/tests/integrations/test_travis.py

```diff
@@ -210,7 +210,6 @@ def test_enable_successfully(
             "Enabling Travis testbeds to push and release 'foo' snaps "
             "to edge channel in series '16'",
             'Acquiring specific authorization information ...',
-            'Login successful.',
```

Comment:
For other reviewers, this change is interesting. Take a look that @filibtester moved the file, because it wasn't being run by ./runtests.sh unit.
So this test got stale, and started failing without us noticing.